README.md

```diff
@@ -50,12 +50,10 @@ See [docs.rs](https://docs.rs/chrono/latest/chrono/) for the API reference.
 
 Default features:
 
+* `clock`: Enables reading the local timezone (`Local`), superset of `now`.
+* `now`: Enables reading the system time (`now()`) using the standard library.
+* `std`: Implements `std::error::Error` for error types, superset of `alloc`.
```

Comment:
I'd like to avoid the "superset of" wording, instead using something like "implies" or "also enables". Should we also make explicit that `now` enables `std` here?

Reply:
I think this is in firmly in [Hyrum's Law](https://www.hyrumslaw.com/) territory, and generally will be seen as semver-breaking anyway if we decide to go back on it so might as well document it.

Reply:
Haha, yes that is something to worry about. But I don't think that will be a problem here.

I added a commit "Don't let `now` imply `std` feature". Do you want to have a look?

With it the `std` feature only enables the `Error` trait, `TryFrom<SystemTime>`, and methods that depend on `alloc`.

I had to extract the code from `TryFrom<SystemTime>` into a method on `DateTime<Utc>` to split the `now` feature from `std`.

Reply:
So what functionality does `now` enable when `std` is disabled?

Reply:
Only `Utc::now()`. And the `clock` feature adds `Local::now()`.

Reply:
`Utc::now()` calls `SystemTime::now()`, so it's actually dependent on `std`, right?

Reply:
Currently it does on all platforms.

Reply:
Now I wonder if we shouldn't just remove the `now` feature and fold it into `std`.

Reply:
But on WASM we need other stuff for `now`, right?

Reply:
You are completely right, I forgot. It wouldn't have to depend on std on that platform.